### PR TITLE
feat: tauri updater 

### PR DESCRIPTION
**Pull request by newfla**
The PR includes:
- Added macos14 aarch64 runner
- Release version tag is extracted from tauri.conf.package.version
- Nightly release are now published as pre-release with tauri updater disabled
- Releases published by manual-release have the tauri updater enabled using "https://github.com/fourviere/fourviere-podcast/releases/latest/download/latest.json" as endpoint

You can try to update [Desktop App v0.1.0](https://github.com/newfla/fourviere-podcast/releases/tag/app-v0.1.0) to 0.1.1 where the feed s3 function will crash the application

### CODE REVIEW

File: .github/disable_updater.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  config.tauri.updater.active = false;
+  
+  fs.writeFile(path, JSON.stringify(config, null, 2), (error) => {
+    if (error) {
+      console.log('An error has occurred ', error);
+      return;
+    }
+    console.log('Data written successfully to disk');
+  });
+});
```
Comment (brainrepo) [inline .github/disable_updater.js:19]: we should add a trailing line

**Comment by brainrepo** [comment]
LDTM! Good work. 
I tried in the fork and it works as expected.
Happy to approve when you feel it ready.

**Comment by brainrepo** [comment]
Seems that the version built with the aarm architecture ([Fourviere.Podcast_0.1.1_aarch64.dmg](https://github.com/newfla/fourviere-podcast/releases/download/app-v0.1.1/Fourviere.Podcast_0.1.1_aarch64.dmg)) is damaged. I am trying to run it  with Apple M2 Pro.

**Comment by brainrepo** [comment]
@newfla  Secrets and key are now available in the github secrets area

File: .github/disable_updater.js (hunk 1 above)
Comment (newfla) [reply .github/disable_updater.js]: Done ✅

**Pull request merged**